lib/clients/BaseClient.py

```diff
@@ -17,32 +17,38 @@ class BaseClient:
     def __init__(self, operation_name, configuration, directory_persistent, directory_work_list, poll_delay_time,
                  poll_maximum_time):
         self.OPERATION = operation_name
-        self.GUID = configuration['backup_guid']
+        # Skipping some parameters for blob operation.
+        if operation_name != 'blob_operation':
+            self.GUID = configuration['backup_guid']
+            self.INSTANCE_ID = configuration['instance_id']
+            self.SECRET = configuration['secret']
+            self.JOB_NAME = configuration['job_name']
+            self.tags = {
+                'created_by': 'service-fabrik-backup-restore',
+                'instance_id': self.INSTANCE_ID,
+                'job_name': self.JOB_NAME
+                }
+        else:
+            self.GUID = 'aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa'
```

Comment:
why is such a default required ?

Reply:
We test if the container is accessible.

e.g. https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/blob/b73f081f5df8a5cc200887fe0c5c76a249453851/lib/clients/GcpClient.py#L100

Reply:
Hmmm , but if there are concurrent run's of the blob operation functions then wont this clash and cause issues since we use the same guid for all calls ?

Reply:
One option is to autogenerate uuid or use current timestamp as value of the guid ?

Reply:
Ok. Using current timestamp too, to ensure the uniqueness of the guid.